doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,7 +48,11 @@ This specification is licensed under the
 
 ## Conformance
 
-All the interfaces described by this specification are mandatory and must be implemented in order to be compliant with. 
+This specification describes the "full" compliance level.
+
+However, next iterations of this specification could define optionality
+in some parts and the compliance level (or roles/profiles)
+corresponding to them.
```

Comment:
As mentioned https://github.com/telefonicaid/fiware-orion/issues/2110#issuecomment-217114454, an optional `capability` field in the `/v2` request would define the subset supported by the  non-"full compliance" implementations. 

The list of capability still need to be defined but we could already agree in the current version of the specification on the mechanism for optionality, if we all agree on it. I voted on your comment @fgalan to approve your proposal.

If in a later discussion, if we cannot agree on this list of capability, this will not break anything as it can stay empty. But not stating anything now about optionality would simply "break" the compatibility promise of this version of the specification.